__tests__/shared/keyManager.ts

```diff
@@ -493,75 +506,80 @@ export default (testContext: {
       }
 
       //@ts-ignore
-      const recovered = recoverTypedSignature({data, signature: signature, version: SignTypedDataVersion.V4})
+      const recovered = recoverTypedSignature({
+        data: data as any,
+        signature: signature,
+        version: SignTypedDataVersion.V4,
+      })
       expect(address.toLowerCase()).toEqual(recovered)
     })
 
     it('should sign credential with eth_signTypedData', async () => {
       const msgParams = {
-        "domain": {
-          "chainId": 4,
-          "name": "VerifiableCredential",
-          "version": "1"
+        domain: {
+          chainId: 4,
+          name: 'VerifiableCredential',
+          version: '1',
         },
-        "types": {
-          "CredentialSubject": [
+        types: {
+          CredentialSubject: [
             {
-              "name": "id",
-              "type": "string"
+              name: 'id',
+              type: 'string',
             },
             {
-              "name": "you",
-              "type": "string"
-            }
+              name: 'you',
+              type: 'string',
+            },
           ],
-          "Issuer": [
+          Issuer: [
             {
-              "name": "id",
-              "type": "string"
-            }
+              name: 'id',
+              type: 'string',
+            },
           ],
-          "VerifiableCredential": [
+          VerifiableCredential: [
             {
-              "name": "@context",
-              "type": "string[]"
+              name: '@context',
+              type: 'string[]',
             },
             {
-              "name": "credentialSubject",
-              "type": "CredentialSubject"
+              name: 'credentialSubject',
+              type: 'CredentialSubject',
             },
             {
-              "name": "issuanceDate",
-              "type": "string"
+              name: 'issuanceDate',
+              type: 'string',
             },
             {
-              "name": "issuer",
-              "type": "Issuer"
+              name: 'issuer',
+              type: 'Issuer',
             },
             {
-              "name": "type",
-              "type": "string[]"
-            }
-          ]
+              name: 'type',
+              type: 'string[]',
+            },
+          ],
         },
-        "message": {
-          "issuer": {
-            "id": "did:fake:123"
+        message: {
```

Comment:
@lleifermann looks like there's some code duplication here causing test failures

Reply:
Fixed it. My prettier messed something up i believe.